pkg/controller/knativeserving/openshift/openshift.go

```diff
@@ -559,6 +744,31 @@ func createRoleAndRoleBinding(instance *servingv1alpha1.KnativeServing, namespac
 	return nil
 }
 
+func watchServiceMeshType(c controller.Controller, mgr manager.Manager, obj runtime.Object) error {
+	return c.Watch(&source.Kind{Type: obj},
+		&handler.EnqueueRequestsFromMapFunc{
+			ToRequests: handler.ToRequestsFunc(func(a handler.MapObject) []reconcile.Request {
+				if a.Meta.GetLabels()[ownerName] == "" && a.Meta.GetLabels()[ownerNamespace] == "" {
```

Comment:
I'd do an `||` here. We don't want to do anything if either of these is not set really. Otherwise the request created below will be invalid anyway.